rdtools/soiling.py

```diff
@@ -286,12 +295,11 @@ def _calc_result_df(self, trim=False, max_relative_slope_error=500.0,
             results.next_inferred_start_loss - results.inferred_end_loss,
             0, 1)
 
-        # Don't consider data outside of first and last valid intervals
         if len(results[results.valid]) == 0:
             raise NoValidIntervalError('No valid soiling intervals were found')
-        new_start = results[results.valid].start.iloc[0]
-        new_end = results[results.valid].end.iloc[-1]
```

Comment:
Not clear to me how this change relates to the others.  If it changes the outputs, does it need a changelog entry?

Reply:
Thanks, yes this is an important change. It retains invalid intervals at beginning and end of the dataset